expr/queue.go

```diff
@@ -0,0 +1,68 @@
+package expr
```

Comment:
Could you copy the copyright header into this file please?

Reply:
It's done.